src/libraries/System.Private.CoreLib/src/System/IO/StringReader.cs

```diff
@@ -126,10 +131,10 @@ public override int Read(Span<char> buffer)
 
             if (_s == null)
```

Comment:
Can we store _s into a local and then use that local for the rest of the method, as we do in earlier methods like Peek?

Reply:
resolved